lib/manager/maven/extract.ts

```diff
@@ -137,6 +137,10 @@ function applyProps(
     result.skipReason = SkipReason.VersionPlaceholder;
   }
 
+  if (propSource) {
```

Comment:
Will propSource ever be a duplicate of packageFile?

Reply:
Yes, it can be assigned only from it. Though I can't figure out it's meaning and when it will be different from new `editFile`

Reply:
Let's only include it if it's different to packageFile